src/Api/Controller/Action/ResendGiftCardEmailAction.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Api\Controller\Action;
+
+use Setono\SyliusGiftCardPlugin\EmailManager\GiftCardEmailManagerInterface;
+use Setono\SyliusGiftCardPlugin\Model\GiftCardInterface;
+use Setono\SyliusGiftCardPlugin\Model\OrderInterface;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+final class ResendGiftCardEmailAction
+{
+    private GiftCardEmailManagerInterface $giftCardEmailManager;
+
+    public function __construct(GiftCardEmailManagerInterface $giftCardEmailManager)
+    {
+        $this->giftCardEmailManager = $giftCardEmailManager;
+    }
+
+    public function __invoke(GiftCardInterface $data): Response
+    {
+        $giftCard = $data;
+        if (($order = $giftCard->getOrder()) instanceof OrderInterface) {
```

Comment:
```suggestion
        if (($order = $giftCard->getOrder()) !== null) {
```

I suppose this is the same because the `OrderInterface` is enforced by type hints, right? The same goes for the code below

Reply:
Yeah true. I C/P from existing controller, but I will refactor to that indeed